lib/connections/service.go

```diff
@@ -90,6 +90,7 @@ var tlsVersionNames = map[uint16]string{
 // dialers. Successful connections are handed to the model.
 type Service interface {
 	suture.Service
+	discover.AddressLister
```

Comment:
Hm?

Reply:
This is entirely unrelated change due to complying with the bot. See explanation in the separate PR, to not pollute the diff: #5947